Bottom: cb4c1449d42538be2ce3619af2ac0852f4508f00
Top:    58424267ac79a08264f4a2fdac7780d19de5ad93
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-17 13:12:05 +0000

Refresh of drc-hack-hack-hack

---

diff --git a/src/find.c b/src/find.c
index bd8c934..92f9d6d 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2506,8 +2506,8 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
               line.Point1.Y = v->point[1];
               line.Thickness = Bloat;
               /* Another Bloat is added by IsLineInPolygon, making the correct
-               * 2x Bloat. Ideally we would change it there, but doing so
-               * breaks some other DRC checks which rely on the  behaviour
+               * 2x Bloat. Perhaps we should change it there, but doing so
+               * breaks some other DRC checks which rely on the behaviour
                * in IsLineInPolygon.
                */
               line.Clearance = 0;
